node/lens.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package node
+
+import (
+	"context"
+
+	"github.com/lens-vm/lens/host-go/engine/module"
+
+	"github.com/sourcenetwork/defradb/client"
+	"github.com/sourcenetwork/defradb/internal/lens"
+)
+
+type LensRuntimeType string
+
+const (
+	DefaultLens LensRuntimeType = ""
```

Comment:
todo: please give it a reasonable value of add a documentation why there is only 1 value here and why it has to be empty string.

Reply:
Will do

- [ ] Doc default + map